--- a/src/hid_core/CMakeLists.txt	2026-06-07 03:01:04
+++ b/src/hid_core/CMakeLists.txt	2026-06-07 03:02:39
@@ -9,6 +9,9 @@
     frontend/emulated_console.h
     frontend/emulated_controller.cpp
     frontend/emulated_controller.h
+    frontend/overlay_state.h
+    frontend/overlay_udp.cpp
+    frontend/overlay_udp.h
     frontend/emulated_devices.cpp
     frontend/emulated_devices.h
     frontend/input_converter.cpp
