--- eden_v0.2.1/src/yuzu/configuration/configure_input_advanced.ui	2026-06-07 03:01:04
+++ eden_v0.2.1_patched/src/yuzu/configuration/configure_input_advanced.ui	2026-06-07 23:02:20
@@ -2757,6 +2757,21 @@
                        </property>
                      </widget>
                    </item>
+                   <item row="9" column="0">
+                     <widget class="QCheckBox" name="overlay_enabled">
+                      <property name="text">
+                       <string>Enable overlay input (UDP)</string>
+                      </property>
+                     </widget>
+                   </item>
+                   <item row="9" column="2">
+                     <widget class="QSpinBox" name="overlay_port">
+                      <property name="minimum"><number>1024</number></property>
+                      <property name="maximum"><number>65535</number></property>
+                      <property name="value"><number>26760</number></property>
+                     </widget>
+                   </item>
+
                  </layout>
                </widget>
              </item>
