--- eden_v0.2.1/src/common/settings.h	2026-06-07 03:01:03
+++ eden_v0.2.1_patched/src/common/settings.h	2026-06-07 23:02:20
@@ -709,6 +709,10 @@
                                            Category::Controls};
     Setting<bool> enable_udp_controller{linkage, false, "enable_udp_controller",
                                         Category::Controls};
+
+    // Overlay
+    Setting<bool> overlay_enabled{linkage, false, "overlay_enabled", Category::Overlay};
+    Setting<u16> overlay_port{linkage, 26760, "overlay_port", Category::Overlay};
 
     Setting<bool> pause_tas_on_load{linkage, true, "pause_tas_on_load", Category::Controls};
     Setting<bool> tas_enable{linkage, false, "tas_enable", Category::Controls};
