--- a/src/common/settings.h	2026-06-07 03:01:03
+++ b/src/common/settings.h	2026-06-07 22:50:41
@@ -709,6 +709,10 @@
                                            Category::Controls};
     Setting<bool> enable_udp_controller{linkage, false, "enable_udp_controller",
                                         Category::Controls};
+
+    // Overlay
+    Setting<bool> enable_overlay{linkage, false, "enable_overlay", Category::Overlay};
+    Setting<u16> overlay_port{linkage, 26760, "overlay_port", Category::Overlay};
 
     Setting<bool> pause_tas_on_load{linkage, true, "pause_tas_on_load", Category::Controls};
     Setting<bool> tas_enable{linkage, false, "tas_enable", Category::Controls};
